lib/sync-settings.coffee

```diff
@@ -28,6 +29,8 @@ module.exports =
 
   activate: ->
     # for debug
+    GitHubApi ?= require 'github'
+    PackageManager ?= require './package-manager'
```

Comment:
Can you move these two lines above the debug message, which belongs to the commands below.